packages/documentation/src/partials/frontend-variables.mdx

```diff
@@ -2,11 +2,11 @@ import { LinkOut } from '@interledger/docs-design-system'
 
 <div class="overflow-table wider-column">
 
-| Variable                         | Helm Value Name                        | Default                         | Description                                                                                                                                                                             |
-| -------------------------------- | -------------------------------------- | ------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `GRAPHQL_URL`                    | frontend.serviceUrls.GRAPHQL_URL       | `http://localhost:3001/graphql` | URL for the GraphQL Admin API                                                                                                                                                           |
-| `OPEN_PAYMENTS_URL`              | frontend.serviceUrls.OPEN_PAYMENTS_URL | `http://localhost:3003/`        | Open Payments API Endpoint                                                                                                                                                              |
-| `PORT`                           | frontend.port                          | `3005`                          | Port from which to host the Remix app                                                                                                                                                   |
-| `ENABLE_INSECURE_MESSAGE_COOKIE` |                                        | _undefined_                     | Values of `true`, `t`, `1` will not use a `secure` message cookie which is required for flash messages to work over http. When not set, the secure flag is set according to `NODE_ENV`. |
+| Variable                         | Helm Value Name                        | Default     | Description                                                                                                                                                                             |
```

Comment:
Let's tag @melissahenderson for this PR so she can make docs changes as necessary